src/assets/main.css

```diff
@@ -281,6 +285,14 @@
   .lux-time-datepicker {
     @apply border-[#767676] border-[1px] pl-1;
   }
+
+  .lux-attribution {
```

Comment:
If `lux-attribution` is only used once, can't we achieve the same with tailwind classes within the component?

Reply:
I prefer having all my styles in a css file, but ok, moved to component template if you think it is better like this 🙂

Reply:
Thanks, I'm just worried, we'll and up with a very long CSS file like in the legacy, making styles less readable.